services/app-api/serverless.yml

```diff
@@ -17,7 +17,7 @@ plugins:
   - serverless-iam-helper
   - serverless-s3-bucket-helper
   - serverless-online
-  - serverless-api-client-certificate
```

Comment:
Please make this file match what is in master, we don't need changes in it now that the proxy is moved into its own service